src/app/help/help.component.ts

```diff
@@ -13,6 +13,17 @@ export class HelpComponent {
 
   version: string = 'dev';
 
+  sections = {
```

Comment:
No this `sections` variable. The toggling should be possible with `currentSection` only.